tasklist/client/src/common/tasks/details/TaskStateLoadingText.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {Loading} from '@carbon/react';
+import type {UserTask} from '@vzeta/camunda-api-zod-schemas/8.8';
+import styles from './TaskStateLoadingText.module.scss';
+import {t} from 'i18next';
+
+type TaskStateLoadingTextProps = {taskState: UserTask['state']};
+
+const statusLoadingMessage = {
+  CREATED: null,
+  ASSIGNING: null,
+  COMPLETED: null,
+  CANCELED: null,
+  FAILED: null,
+  UPDATING: t('Please wait while the task is being updated...'),
+  CANCELING: t('The task is being cancelled. Please wait...'),
+  COMPLETING: t('Completing...'),
+};
+
+const TaskStateLoadingText: React.FC<TaskStateLoadingTextProps> = ({
+  taskState,
+}) => {
+  if (statusLoadingMessage[taskState] === null) return null;
```

Comment:
Nitpick but this is more consistent with the codebase

```suggestion
  if (statusLoadingMessage[taskState] === null) {
    return null;
  }
```